.gitignore

```diff
@@ -38,4 +38,9 @@ terraform
 # Ignore the Go vendor directory, superseded by proxy.golang.org and go.{mod,sum}
 vendor
 
+# ignore binaries in the root dir
 terraform-provider-iterative
+leo-*
```

Comment:
```suggestion
leo*
```
Just in case?

Reply:
now just `leo` since we dropped suffixes